src/main/java/com/roommatematcher/enterprise/service/ProfileServiceStub.java

```diff
@@ -35,6 +36,11 @@ public Profile fetchBySmokingPreference(Boolean smokingPreference) {
         return profile;
     }
 
+    @Override
+    public Profile save(Profile profile) throws Exception {
+        return profileDAO.save(profile);
+    }
+
```

Comment:
Is this used anywhere?  I don't see a reference to it.

This appears to be an incomplete feature.  In a code review, focus on reducing technical debt of existing code.  Avoid adding new, incomplete functionality .
1) Only features that meet the Definition of Done should be merged to Master.
2) Unused code is clutter, which increases technical debt.